source/_posts/2024-10-02-connect-zbt1-issue-and-replacement.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Home Assistant Connect ZBT-1 issue and replacement"
+description: "Through extensive testing, we’ve found a fault, but we’re setting up a replacement program."
+date: 2024-10-02 00:01:01
+date_formatted: "October 2, 2024"
+author: Andy Gill
+comments: true
+categories: Public-Service-Announcement
+og_image: /images/blog/2024-10-zbt1-issue/art.jpg'
+---
+<img src='/images/blog/2024-10-zbt1-issue/art.jpg' alt="Home Assistant Connect ZBT-1 issue and replacement">
+
+We've found an issue in Home Assistant Connect ZBT-1 that has affected a small number of customers, which appears to be the result of a faulty part incorporated in the most recent batch of ZBT-1 devices. We want to make sure everyone can get a replacement ZBT-1 and ensure they are covered far into the future.
+
+Though Home Assistant SkyConnect is identical to the ZBT-1 under the hood, it is not affected.  The design is not what's at fault, but instead, a defective voltage regulator that was used in this specific batch that is causing the issue. When a ZBT-1 is inserted into certain USB-A ports, the voltage regulator can fail, fully bricking the device. We used high-quality manufacturing and quality control in this and previous batches. We are undertaking an investigation to find out how this could happen and most importantly, to avoid a repeat of anything remotely similar in the future.
```

Comment:
```suggestion
Though Home Assistant SkyConnect is identical to the ZBT-1 under the hood, it is not affected. The design is not what's at fault, but instead, a defective voltage regulator that was used in this specific batch that is causing the issue. When a ZBT-1 is inserted into certain USB-A ports, the voltage regulator can fail, fully bricking the device. We used high-quality manufacturing and quality control in this and previous batches. We are undertaking an investigation to find out how this could happen and most importantly, to avoid a repeat of anything remotely similar in the future.
```